R/enumpart.R

```diff
@@ -21,9 +22,9 @@ redist.init.enumpart <- function(){
     makecontent[7] <-"\tg++ enumpart.cpp SAPPOROBDD/bddc.o SAPPOROBDD/BDD.o SAPPOROBDD/ZBDD.o -o enumpart -I$(TDZDD_DIR) -std=c++11 -O3 -DB_64 -DNDEBUG -lpsapi"
     writeLines(text = makecontent, con = system.file('enumpart/Makefile', package = 'redist'))
   }
-  
+
   servr::make(dir = system.file('enumpart', package = 'redist'), verbose = FALSE)
-  sys::exec_wait('python', args= c('-m', 'pip', 'install', 'networkx', '--user'))
+  sys::exec_wait('python3', args= c('-m', 'pip', 'install', 'networkx', '--user'))
```

Comment:
Does specifying python3 rather than python have a particular effect worth noting? I really need to rewrite this in R before python code needs to be maintained.

Reply:
I believe it wasn't possible to get this working if I didn't use python3... but if it works on your computer then maybe we leave as python

Reply:
That's fine. It would seem to me that your version is better, as people who like using python might have python2 before python3 on their path, which is likely the source of the issue. Networkx seems to only be available for newer pythons.